.github/workflows/kubevirtci-tag-bumper.yaml

```diff
@@ -0,0 +1,53 @@
+name: Bump KubevirtCI Tag
+
+on:
+  schedule:
+    - cron: '0 0 1 1,5,9 *'  # 00:00 on the 1st of Jan, May, Sep
```

Comment:
please change to 3M, worth to change the comment to say it is 3M

Reply:
DONE